otelcol/flags.go

```diff
@@ -49,6 +63,10 @@ func flags(reg *featuregate.Registry) *flag.FlagSet {
 			return nil
 		})
 
+	mergePaths := new(mergePathsValue)
+
+	flagSet.Var(mergePaths, mergePathsAppendFlag, "Specify config paths you want merged")
```

Comment:
```suggestion
	flagSet.Var(mergePaths, mergePathsAppendFlag, "Specify paths in the config where lists should be merged instead of overwritten. `*` specifies that all paths with lists should be merged.")
```